src/gridfs/upload.ts

```diff
@@ -97,6 +104,8 @@ export class GridFSBucketWriteStream extends Writable {
    * ```
    */
   gridFSFile: GridFSFile | null = null;
+  /** @internal */
+  timeoutContext?: CSOTTimeoutContext;
```

Comment:
Since the majority of the implementation of the class is done in floating functions that take in an instance of a `GridFSUploadStream`, this either has to remain public or we can refactor to turn those floating functions into private methods. I'm open to either.

Reply:
Oh right, no worries leaving this 'as is' is fine